e2etest/newe2e_resource_manager_azstorage.go

```diff
@@ -30,47 +32,88 @@ func (acct *AzureAccountResourceManager) ApplySAS(URI string, loc common.Locatio
 		return URI
 	}
 
-	sasVals := opts.AzureOpts.SASValues
-	if sasVals == nil {
+	var sasVals GenericSignatureValues
+	if opts.AzureOpts.SASValues == nil {
+		// Default to account level SAS to cover all our bases
```

Comment:
(up to you) we could generate with reasonable defaults on a generic service signature values if we're >container. We can pull the container name and object name, and already do. This would be more "secure" esp. if we delete the container later, which we do.

But, all the resources for the framework disappear at the end anyway, so, it's equally as "secure"

Reply:
leave default to account sas